docs/sources/ocimetrics.md

```diff
@@ -0,0 +1,73 @@
+# Oracle Cloud Infrastructure Metrics (OCIMetrics) source for knative eventing
+
+This event source collects metrics data from the [Oracle Cloud](https://cloud.oracle.com).
+
+## Prerequisites
+
+An Oracle Cloud account with permissions to inspect and read metrics for the
+Oracle Cloud Infrastructure (OCI) compartment is required.  In addition, the
+account must have the account API secret added.
+
+For additional information on how to create an API key and associate it with
+your Oracle Cloud user, go to: https://docs.cloud.oracle.com/en-us/iaas/Content/API/Concepts/apisigningkey.htm#Required_Keys_and_OCID
+
+### Adding the Oracle Cloud Secret
+
+Three pieces of information are required for the Oracle Cloud:
+1. API Private Key used for signing the request
+1. API Private Key passphrase to decrypt the key
+1. API Key's fingerprint to identify which key to use on the Oracle Cloud end
+
+From the Navigation Menu, click `Secrets`, then click the `ADD SECRET` button on
```

Comment:
Shouldn't we reference the Secrets doc page instead, and tell the user to create a secret of type Oracle by referring to it?

Reply:
You're right in this case. I'll remove the entry.